kore/src/main/java/org/kframework/builtin/Hooks.java

```diff
@@ -25,6 +25,7 @@
     public static final String STRING = "STRING";
     public static final String SUBSTITUTION = "SUBSTITUTION";
     public static final String UNIFICATION = "UNIFICATION";
+    public static final String JSON = "JSON";
 
-    public static final Set<String> namespaces = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(ARRAY, BOOL, BUFFER, BYTES, FFI, FLOAT, INT, IO, KEQUAL, KREFLECTION, LIST, MAP, MINT, SET, STRING, SUBSTITUTION, UNIFICATION)));
+    public static final Set<String> namespaces = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(ARRAY, BOOL, BUFFER, BYTES, FFI, FLOAT, INT, IO, KEQUAL, KREFLECTION, LIST, MAP, MINT, SET, STRING, SUBSTITUTION, UNIFICATION, JSON)));
```

Comment:
This PR doesn't seem to add anything for JSON parsing handling, should we wait for the JSON module to add this?

Reply:
Whoops, that's my mistake, it's not intended to be part of this pr. I will fix.